jingyued/Redpandas
# Chirp feature/joyce/bamboo 56/display likelist

Create LikeList Component and display likedList with custom pipe:
- Create LikeList Component according to your UI design
- LikeList can be stored locally (it will become empty when refresh the page)
- When user click "Like List" button, it will display/hide the LikeList Component
- The custom pipe should take string as input, and output only the first 20 characters. If the input is more than 20 characters, display "..." at the end of the converted string

How to test: 
- run `ng serve`
- like some chirrups in home page to see it displayed in the liked page
- eg. if you like this post in the home page, you will see it's content truncated in the liked page
- 
<img width="808" alt="Screenshot 2024-05-03 at 12 52 52 AM" src="https://github.com/jingyued/Redpandas/assets/113967795/07cb4e08-a536-4b59-bed3-51dc58a94bec">

<img width="434" alt="Screenshot 2024-05-03 at 12 49 52 AM" src="https://github.com/jingyued/Redpandas/assets/113967795/7605dfed-ece2-42b1-99c9-e39442d4fbc6">
